src/test/scala/mesosphere/marathon/storage/migration/MigrationTo160Test.scala

```diff
@@ -0,0 +1,129 @@
+package mesosphere.marathon
+package storage.migration
+
+import java.time.OffsetDateTime
+
+import akka.Done
+import akka.http.scaladsl.unmarshalling.Unmarshaller
+import akka.stream.scaladsl.Source
+import akka.stream.{ ActorMaterializer, Materializer }
+import com.typesafe.scalalogging.StrictLogging
+import mesosphere.AkkaUnitTest
+import mesosphere.marathon.core.condition.Condition
+import mesosphere.marathon.core.instance.Instance.Id
+import mesosphere.marathon.core.instance.{ Instance, Reservation, TestInstanceBuilder }
+import mesosphere.marathon.core.storage.store.{ IdResolver, PersistenceStore }
+import mesosphere.marathon.core.storage.store.impl.zk.{ ZkId, ZkPersistenceStore, ZkSerialized }
+import mesosphere.marathon.core.task.Task
+import mesosphere.marathon.core.task.state.NetworkInfo
+import mesosphere.marathon.state._
+import mesosphere.marathon.storage.migration.MigrationTo146.Environment
+import mesosphere.marathon.storage.repository.InstanceRepository
+import mesosphere.marathon.test.GroupCreation
+import org.apache.mesos
+import org.apache.mesos.Protos.NetworkInfo.Protocol
+import play.api.libs.json.{ JsValue, Json }
+import play.api.libs.json._
+
+import scala.concurrent.duration._
+import scala.concurrent.{ ExecutionContextExecutor, Future }
+
+class MigrationTo160Test extends AkkaUnitTest with GroupCreation with StrictLogging {
+
+  "Migration to 1.6.0" should {
+    "do nothing if env var is not configured" in new Fixture {
```

Comment:
The comment is misleading. I assume the idea is that nothing should be done, if there are no apps with PVs. This test covers the case when there is nothing stored ZK. I think it would be nice to cover the case in which nothing should be done, if there are apps without PVs and pods stored in ZK. What do you think?